src/styles.scss

```diff
@@ -52,12 +49,38 @@ $primary-color: #46a9ff;
 $black-color: #4c4c4c;
 $blue-gradient: #1AC6FC;
 $grey-color: #bababa;
+$light-grey-color:#e0e0e0;
 $regular-font: 'SF-ui-text-regular';
 $light-font: 'SF-ui-text-light';
 $italic-font: 'SF-ui-text-italic';
-$border-color: #e5e5e5;
+$border-color : #e5e5e5;
+$green-color: rgb(1, 116, 1);
+$white-color: #fff;
+$background-color :#f2f2f2;
+$button-background-color: #e5f2fe;
+$dark-blue:#1d73f0;
+
+:root{
+--primary-color:#46a9ff;
```

Comment:
--blue-gradient: $blue-gradient

Implement in this way so that we need to change the variable in one variable for changing the theme.